docs/portal/team-members.md

```diff
@@ -60,14 +70,23 @@ To re-send the invitation for a team member, complete the following steps:
 
 ## Edit Information for a Team Member
 
-Admin users can update any of the values: First Name, Last Name, Email, Job Function, or Role.
+Notes re role:
+- Users with the Admin role can update any of these values, for any team member in their participant organization.
+- Users with the Operations role have read-only access to the Team Members page, but cannot change the information.
 
 To edit information for a team member, complete the following steps:
 
 1. Log in to your UID2 Portal account.
 1. At the top right, click the down arrow and choose **Manage Team Members**.
 1. Find the team member on the list and then, in the Actions column, click ![the Edit icon](images/icon-pencil-solid.png) (the Edit icon).
-1. Update the information, and then save the changes.
+1. Update any of these values:
+   - First Name
+   - Last Name
+   - Job Function
+   - Role: For information about roles, see [User Roles](#user-roles).
+   
+   **Note**: You cannot update the email for an existing team member. If you need to change the email, add a new team member using the new email. This ensures that the email verification process is correctly followed.
```

Comment:
I'd add something about deleting a team member with an old email as well in the process of changing the email.

Reply:
Great call. Done.